packages/operation/src/evm/deploy/funding.ts

```diff
@@ -5,21 +5,23 @@ export async function fundProviderAccounts(state: State): Promise<State> {
   for (const apiProviderName of Object.keys(state.apiProvidersByName)) {
     const apiProvider = state.apiProvidersByName[apiProviderName];
     // Ensure that the API provider address has enough ETH to create the onchain provider
-    await state.deployer.sendTransaction({
+    const tx = await state.deployer.sendTransaction({
       to: apiProvider.address,
       value: ethers.utils.parseEther('5'),
     });
+    await tx.wait();
```

Comment:
These `wait` calls are not strictly necessary but better to just have them to avoid any weird behaviour I think